mp/src/gameui/OptionsSubKeyboard.cpp

```diff
@@ -483,52 +412,6 @@ void COptionsSubKeyboard::UnbindKey( const char *key )
     engine->ClientCmd_Unrestricted(UTIL_va("unbind \"%s\"\n", key));
```

Comment:
`UTIL_va` should be replace with `CFmtStr`

Reply:
And `UTIL_Parse` can be removed with `engine->parse` being called directly instead.